weather

```diff
@@ -0,0 +1,16 @@
+api_key = 'eb3f523c4bedada94d5846340c5dbde7'
+one_call = 'https://api.openweathermap.org/data/2.5/onecall?lat=35.6870&lon=-105.9378&exclude=hour,daily&units=metric'
+headers = {'Content-Type': }
```

Comment:
Are these headers used? It seems to be empty for the entire file, only the key 'Content-Type' is there?